fetch_usage_from_ccdb.sh

```diff
@@ -51,7 +51,7 @@ PSQL_URL=postgresql://localhost:3306/cloud_controller
 bosh \
     ssh -d cf_databases ccdb/0 "
     sudo -u vcap -i \
-    /var/vcap/packages/postgres-9.6.4/bin/psql ${PSQL_URL} \
+    /var/vcap/packages/postgres-9.6.8/bin/psql ${PSQL_URL} \
```

Comment:
Auto-detecting the pg version is surprisingly non-trivial because of the way that `bosh ssh` handles quoting. Variables either get expanded on the local side of the connection, or passed through as a single quoted string by bosh and then are not evaluated when the script is run at the remote end. The same thing happens with wild-card chars. This is very much a quirk in the behavior of `bosh ssh` versus a normal ssh session. A couple of us (Hector and myself) have already had a go at making the script smarter and both of us independently came to the conclusion that it was more trouble than it was worth.  Reworking the whole of this script to work around the limitations of `bosh ssh` would be nice, but for expediency I'm going to merge this PR as it gets us past the current problem and will let our usage reports generate correctly once again.